.eslintrc.js

```diff
@@ -32,5 +32,6 @@ module.exports = {
     'dist/',
     'docs/',
     '.yarn/',
+    'scripts/query/',
```

Comment:
Why is this ignored?

Reply:
The Query uses `fs`, `path` and `process` to manage local file writes, reads and env vars.
This violates the eslint rules. So we can either relax the eslint rules or we can ignore it for the Query files.
Please let me know if there is another option.

Reply:
If you change
```
files: ['*.js', 'scripts/*.ts'],
```
to
```
files: ['*.js', 'scripts/**/*.ts'],
```
a little further up in the config, and remove this line, most of the lint errors will be fixed.

Reply:
Thanks.
Fixed as per your suggestion. 👍